docs/hotkeys.md

```diff
@@ -64,6 +64,7 @@
 |View current message in browser (from message information)|<kbd>v</kbd>|
 |Show/hide full rendered message (from message information)|<kbd>f</kbd>|
 |Show/hide full raw message (from message information)|<kbd>r</kbd>|
+|Copy message content to clipboard|<kbd>c</kbd>|
```

Comment:
This doesn't have any `(from ...)` suffix like the others, which could imply that it works just when a message is selected.

Reply:
Missed that text. Adding `(from message information)`. Or should I add something like `(from raw message view)`? Although it might be confusing for new users since they might not know where raw message view is.